### ISSUE TITLE
PilotNet-LIF training code?

### ISSUE THREAD

**Issue by R-Gaurav** [lava-nc/lava-dl#367]
### User story
I couldn't find the **training** code for PilotNet-LIF example. 

I do see a [PilotNet-SDNN training](https://github.com/lava-nc/lava-dl/blob/main/tutorials/lava/lib/dl/slayer/pilotnet/train.ipynb) example (and it's corresponding [netx inference](https://github.com/lava-nc/lava-dl/blob/main/tutorials/lava/lib/dl/netx/pilotnet_sdnn/run.ipynb)). However, despite seeing a [PilotNet-LIF netx inference](https://github.com/lava-nc/lava-dl/blob/main/tutorials/lava/lib/dl/netx/pilotnet_snn/run.ipynb), I was unable to find it's corresponding training code.

### Conditions of satisfaction
Please provide a training notebook for the PilotNet-LIF example.

### Acceptance tests
I am particularly interested in the regression part of PilotNet-LIF where the output neuron's voltage state is used to decode the steering angle. How would someone use the output neuron's voltage to compute training loss in a regression setting (in a `slayer` framework)? What would be the PilotNet-LIF's network layers?
